osmtogtfs/gtfs/gtfs_writer.py

```diff
@@ -98,10 +98,9 @@ def add_frequencies(self, frequencies):
     def add_file(self, name, path):
         self._files[name] = path
 
-    def write_zipped(self, filepath, compress=True):
+    def write_zipped(self, filepath):
         """Write the GTFS feed in the given file."""
-        compression = zipfile.ZIP_LZMA if compress else zipfile.ZIP_STORED
-        with zipfile.ZipFile(filepath, mode='w', compression=compression) as zfile:
+        with zipfile.ZipFile(filepath, mode='w', compression=zipfile.ZIP_DEFLATED) as zfile:
```

Comment:
I just found out that deflate is also the algorithm used in Java

Reply:
Apparently it is very common, i.e. universal!